x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/create_rules_bulk_route.ts

```diff
@@ -85,9 +81,9 @@ export const createCreateRulesBulkRoute = (server: ServerFacade): Hapi.ServerRou
             } = payloadRule;
             const ruleIdOrUuid = ruleId ?? uuid.v4();
             try {
-              const finalIndex = outputIndex != null ? outputIndex : getIndex(request, server);
-              const callWithRequest = callWithRequestFactory(request, server);
-              const indexExists = await getIndexExists(callWithRequest, finalIndex);
+              const finalIndex =
+                outputIndex != null ? outputIndex : getIndex(spacesClient.getSpaceId, config);
```

Comment:
optional: I think we can use the fancy smancy `??` here? Otherwise still looks good, sorry you're refactoring my code. I have old habits I am trying to stop doing such as this one.

Reply:
@FrankHassanabad In the course of making these changes I think I might've found a bug: https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/import_rules_route.ts#L149 does not seem to respect an imported rule's output index.